.gitignore

```diff
@@ -3,5 +3,6 @@
 /build/
 .idea/
 arisa.yml
```

Comment:
Is this `.gitignore` entry still needed then?

Reply:
I kept it intentionally, so that no one accidentally commits credentials from an old *.yml.